user_docs/en/changes.t2t

```diff
@@ -19,6 +19,9 @@ There's also been bug fixes for the Add-on Store, Microsoft Office, Microsoft Ed
 
 == New Features ==
 - Enhanced sound management:
+  - New "audio" panel in settings. (#15472)
+    - Options to control audio output and ducking mode have been relocated to audio settings. They will be removed from the "select synthesizer" dialog in 2024.1. (#15486)
+    -

```

Comment:
Can you move this to the line "The settings to change audio output device and toggle audio ducking have been moved to the new Audio settings panel from the Select Synthesizer dialog. (#8711)" in changes?

Perhaps just adding the clarification "They will be removed from the "select synthesizer" dialog in 2024.1" as a new line in the same bullet point.